<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGCSwarmControl</class>
 <widget class="QWidget" name="QGCSwarmControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>682</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="continueAllButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Return2Start">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to start</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>81</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="scenarioLaunch">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set scenario</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="radiusSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="numVhcSpinBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string># Vhc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Circle Radius</string>
   </property>
  </widget>
  <widget class="QPushButton" name="autoLanding">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto landing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="autoTakeoff">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto takeoff</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="altitudeSpinBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Altitude</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoContinueCheckBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto continue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startLogging">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Logging</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopLogging">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Logging</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxParam1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="setParameters">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="armButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Arm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disarmButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Disarm</string>
   </property>
  </widget>
  <widget class="QListWidget" name="remoteList">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>81</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Unselect all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remoteButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Set control</string>
   </property>
  </widget>
  <widget class="QComboBox" name="modeComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>116</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Select MAV operation mode</string>
   </property>
   <property name="statusTip">
    <string>Select MAV operation mode</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="avoidanceBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Collision avoidance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setHomeButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Set home</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxParam2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxParam3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxParam4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxParam5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxParam6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxParam7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="wptReachedLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string># of MAV arrived:</string>
   </property>
  </widget>
  <widget class="QLabel" name="wptReachedValue">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>600</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxScenario">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Circle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Circle uniform</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Stream</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Swarm</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxStrategy">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ORCA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Human</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PFM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Flocking</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="strategyLaunch">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set strategy</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../qgroundcontrol.qrc"/>
 </resources>
 <connections/>
</ui>
